package.json

```diff
@@ -53,7 +53,7 @@
     "redux-thunk": "^2.2.0"
   },
   "dependencies": {
-    "axios": "^0.17.1",
+    "axios": "^0.19.0",
```

Comment:
You may be able to just remove this dependency (and also `axios-mock-adapter` as well) since it's actually [provided by foreman](https://github.com/theforeman/foreman/blob/develop/package.json#L72).  Although this change may also require changes to tests, I'm not sure.

Reply:
@waldenraines  react tests did fail when i removed it, thoughts?

Reply:
I'd have to know why they failed.  It probably means that it needs to be included differently in the test.

Reply:
Disregard the above ;)